sql/database.sql

```diff
@@ -1653,6 +1653,7 @@ CREATE TABLE `facility` (
   `oid` VARCHAR(255) NOT NULL DEFAULT '' COMMENT 'HIEs CCDA and FHIR an OID is required/wanted',
   `iban` varchar(50) default NULL,
   `info` TEXT,
+  `weno_id` VARCHAR(10) NULL DEFAULT NULL,
```

Comment:
remove the first NULL

Reply:
it is always fun adding a new facility since then have to incorporate it into the below insert of the facility (or it breaks the installation). Just add a new item to the end of insert facility line below (use null in it)

Reply:
This was changed and pushed up.
![image](https://user-images.githubusercontent.com/834958/104451616-cd5bac00-556f-11eb-92c7-551de1d60f3b.png)

Reply:
what i meant to say is that the line below this block of code:
```
INSERT INTO `facility` VALUES (3, NULL, 'Your Clinic Name Here', '000-000-0000', '000-000-0000', '', '', '', '', '', '', NULL, NULL, 1, 1, 0, NULL, '', '', '', '', '', '','#99FFFF','0', '', '1', '', '', '', '', '', '', '', '');
```
Needs to be changed to (added a NULL at the end):
```
INSERT INTO `facility` VALUES (3, NULL, 'Your Clinic Name Here', '000-000-0000', '000-000-0000', '', '', '', '', '', '', NULL, NULL, 1, 1, 0, NULL, '', '', '', '', '', '','#99FFFF','0', '', '1', '', '', '', '', '', '', '', '', NULL);
```